netlify.toml

```diff
@@ -385,7 +385,7 @@
 
 [[redirects]]
```

Comment:
we'll need to add net new redirects to forward "/guides/xxx-integration" to "/guides/integration/xxx"